mk/run-test.sh

```diff
@@ -8,11 +8,12 @@ yellow=""
 normal=""
 
 test=$1
+init=${2-}
 
 dir="$(dirname "${BASH_SOURCE[0]}")"
 source "$dir/common-test.sh"
 
-post_run_msg="ran test $test..."
+post_run_msg="ran test $test_name..."
```

Comment:
Having the whole file name makes it easier to understand for newcomers.

```suggestion
post_run_msg="ran test $test..."
```